core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-history/dot-container-history.component.scss

```diff
@@ -0,0 +1,4 @@
+dot-portlet-box {
+    height: 100vh;
```

Comment:
Don't we have some padding and header that could break with this?

Reply:
@zulqarnainvd